apps/widgets/src/features/AdminDashboard/WidgetInfo.tsx

```diff
@@ -0,0 +1,116 @@
+import type { FC } from 'react';
+import { StyleSheet } from 'react-native';
+import { type Maybe, type Widget, WidgetStatus } from '@walless/graphql';
+import { Button, Text, View } from '@walless/gui';
+import Image from 'next/image';
+
+interface RowProps {
+	title: string;
+	value: Maybe<string | undefined>;
+}
+
+const RowText: FC<RowProps> = ({ title, value }) => (
+	<View style={styles.row}>
+		<Text style={styles.title}>{title}</Text>
+		{value ? (
+			<Text style={styles.value}>{value}</Text>
+		) : (
+			<Text style={styles.value}>No value</Text>
+		)}
+	</View>
+);
+
+const RowImage: FC<RowProps> = ({ title, value }) => (
+	<View style={[styles.row, { flexDirection: 'column' }]}>
+		<Text style={styles.title}>{title}</Text>
+		{value ? (
+			<Image src={value} alt={title} width={100} height={100} />
+		) : (
+			<Text style={styles.value}>No value</Text>
+		)}
```

Comment:
I guess, use an actual "empty" Image instead of "No value" text, then we don't have to do conditional blocks.
Just one single `<Image />` tag with dynamic/conditional `src` attribute.